### ISSUE TITLE
workload as nightshift

### ISSUE THREAD

**Issue by MrGamy** [engelsystem/engelsystem#710]
Currently workloads are not counted towards shirtscore, but doubling nightly workloads is bad for the other counters attached to the hours worked. The ability to count workloads as nightshifts would be great. -> Checkbox?

**Comment by rugk** [comment]
I guess this issue here is about the problem that it shows you only your planned hours without double-counting nightshiffs…?

They are only doubke-counted in your T-Shirt score™.

**Comment by rugk** [comment]
If it is about this, then I agree this is a big usability problem. :wink:

**Comment by MrGamy** [comment]
No, "workloads" - which are manually added - are never counted as nightshifts and therefore also not double-counted in the T-Shirt score™. There is currently no option to mark those workloads as nightshifts.

**Comment by rugk** [comment]
Ah okay, so workloads are something else, okay, then this is not what I said… I'll create a new issue: https://github.com/engelsystem/engelsystem/issues/715

**Comment by olebittner** [comment]
I think this issue is resolved or is there more work to be done here?

**Comment by MrGamy** [comment]
resolved by [#1502](https://github.com/engelsystem/engelsystem/pull/1502)
